src/REmatchEngine.cpp

```diff
@@ -214,13 +205,8 @@ void REmatch2AutomataEngine::compile(const std::vector<Rule>& rules,
       opt |= REMATCH_MOD_DOTALL;
     mods.push_back(opt);
   }
-  auto matcher =
-      rematch2_compile(ids.data(), exps.data(), mods.data(), ids.size(), reduce
-#ifdef USE_TURBO
-                       ,
-                       turbo
-#endif
-                       );
+  auto matcher = rematch2_compile(ids.data(), exps.data(), mods.data(),
```

Comment:
Shouldn't this be rematch2_compile_with_shortcuts()?

Reply:
I am working on it~